src/Tribe/Admin/Help_Page.php

```diff
@@ -29,7 +57,15 @@ public static function instance() {
 	 * @return bool
 	 */
 	public function is_current_page() {
-		return Tribe__Admin__Helpers::instance()->is_screen( 'tribe_events_page_tribe-help' ) || Tribe__Admin__Helpers::instance()->is_screen( 'settings_page_tribe-common-help-network' );
+		global $current_screen;
+
+		// @todo: Make this filterable and add it from plugins.
```

Comment:
If still required, then just filter the value and add the filter doc-block, if not, remove.